TwinCAT3.gitignore

```diff
@@ -8,6 +8,9 @@
 *.tclrs
 *.compiled-library
 *.compileinfo
+# Don't include the tmc-file rule if either of the following is true:
```

Comment:
```suggestion
# uncomment if you are using TwinCAT3 with a general purpose programming language (C++, C#, ...)
```

Reply:
General purpose languages have nothing to do with TMC-files, other than specifically IEC61131-3 (PLC languages) and specifically C++.